config/locales/en.yml

```diff
@@ -81,14 +81,64 @@ en:
         # API spec for Simple Server
         ## Sync APIs
         This API spec documents the endpoints that the devices (that are offline to varying levels) will use to sync data. The sync end points will send and receive bulk (a list of) entities. Both sending and receiving can be batched with configurable batch-sizes to accommodate low network bandwidth situations.
+
         ## Nesting resources
         The APIs have been designed to provide an optimal balance between accuracy and simplicity. Some of the APIs (patients) will be nested, and some other (blood pressures) will be flat.
-        ## Authentication
-        TODO
 
         ## Sync Mechanism
         Refer to the [related ADR](https://github.com/simpledotorg/simple-server/blob/master/doc/arch/001-synchronization.md).
 
+        ## API Authentication
+        There are two types of authentication available to access different parts of the Simple API.
+        - User Authentication - For medical professionals using the Simple App. Grants access to most of the Simple API
+          to read and write data for communities of patients.
+        - Patient authentication - For individual patients. Grants access to read a patient's own data.
+
+        ### User Authentication
+
+        A Simple client can make authenticated requests to the Simple API on behalf of a medical professional with the
+        following credentials:
+        - API Access Token
+        - User ID
+        - Facility ID
+
+        In order to obtain these credentials, a Simple client can authenticate a user with the following steps. The
+        client will need the user's registered phone number to perform the authentication.
+        1. Submit the users phone number to the [`POST /v4/users/find` endpoint](#tag/User/paths/~1users~1find/post). If
+        a user exists with this phone number, the response will return their id.
+        2. Submit the user's id retreived in the previous step, along with the user's password to the
+        [`POST v4/users/activate` endpoint](#tag/User/paths/~1users~1activate/post).
+        This will validate the user's password, and trigger an OTP to be sent to the user's phone number.
+        3. Submit the user's id, their password, and the OTP (received after the previous step) to the
+        [`POST v3/login` endpoint](#tag/User-Login/paths/~1login/post). If the submitted otp and password are valid,
```

Comment:
How long is the access token valid for?  Does a client need to login on a some periodic basis to refresh the token?

Reply:
It's valid until the user signs into another device. I'll add explicit info.